readthedocs/rtd_tests/tests/test_project_views.py

```diff
@@ -414,6 +414,22 @@ def test_delete_project(self):
                 args=[(project.doc_path,)],
             )
 
+    def test_delete_superproject(self):
+        super_proj = get(Project, slug='pip', users=[self.user])
+        sub_proj = get(Project, slug='test-sub-project', users=[self.user])
+
+        self.assertFalse(super_proj.subprojects.all().exists())
+        super_proj.add_subproject(sub_proj)
+
+        response = self.client.get('/dashboard/pip/delete/')
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(
```

Comment:
assertContains is a shorcut for this https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains

Reply:
Thank you.
This was a nice shortcut.
I have updated the PR.

Reply:
status code is also checked by assertContains ;)

Reply:
Default is already 200.
Should we write it here explictly?

Reply:
Default is fine